### ISSUE TITLE
Add text prop and deprecate textForPercentage/classForPercentage props

### ISSUE THREAD

**Issue by iqnivek** [iqnivek/react-circular-progressbar#29]
This is per discussion in #24.

Add warning messages for textForPercentage/classForPercentage, and fully remove them in a future major version.

**Comment by angelcustodio** [comment]
I think `classForPercentage` is very useful for styling purposes.
Imagine you want to color the progress differently while it changes from a value to another to give some visual information in addition of the percentage.

PS: The percentage should be optional also having the possibility to pass an element as a children to render it there. Cool, huh? :)

PS2: Great job :D

**Comment by iqnivek** [comment]
Thanks @angelcustodio! :D

> Imagine you want to color the progress differently while it changes from a value to another to give some visual information in addition of the percentage.

Yeah, I'm not intending to remove that functionality! The intent is rather to shift the logic from being a `classForPercentage` function to using `className` directly. For instance, if you were using `classForPercentage` like this before:

```js
function getClassForPercentage(percentage) {
  if (percentage < 50) {
    return 'some-class';
  }
  return 'some-other-class';
}

<CircularProgressbar percentage={someValue} classForPercentage={getClassForPercentage} />
```

...you could rewrite it as this:

```js
function getClassForPercentage(percentage) {
  if (percentage < 50) {
    return 'some-class';
  }
  return 'some-other-class';
}

<CircularProgressbar percentage={someValue} className={getClassForPercentage(someValue)} />
```

You don't have to change much, and it seems conceptually easier to grok (from feedback from jedwards1211 in #24). Does this address your concern?

> The percentage should be optional also having the possibility to pass an element as a children to render it there.

I don't quite follow what you mean exactly - could you give me an example?

**Comment by iqnivek** [comment]
Released in version 1.0.0.
